DataFormats/HeavyIonEvent/interface/HFFilterStruct.h

```diff
@@ -0,0 +1,13 @@
+#ifndef DataFormats_HeavyIonEvent_HFFilterStruct_H
+#define DataFormats_HeavyIonEvent_HFFilterStruct_H
+
+namespace pat {
+  struct HFFilterStruct {
+    int numMinHFTowers2;
```

Comment:
these could be unsigned shorts?

Reply:
implemented in commit 78b0a76436417538aefafc8a492dd0314831b794